etor/src/main/java/gov/hhs/cdc/trustedintermediary/plugin/path/Hl7FhirMappingPath.java

```diff
@@ -7,11 +7,16 @@
  * defines the HL7 field names.
  */
 public enum Hl7FhirMappingPath {
-    PLACER_ORDER_NUMBER(
+    PLACER_ORDER_NUMBER_ORC_2(
             "ORC.2",
             """
      Bundle.entry.resource.ofType(ServiceRequest).identifier.where(type.coding.code = 'PLAC').value
      """),
+    PLACER_ORDER_NUMBER_OBR_2(
```

Comment:
Is this path being used?

Reply:
I've got _no usages_ coming up too; Is this a question of whether we need this now or will need it?

Reply:
I don't think it is right now, but we're also using this FHIR paths as reference. Even if not used, it's useful for us to document all the HL7 FHIR paths

Reply:
I’ll approve the PR, but I would recommend that we add paths as and when they are needed. This way, we can ensure that every documented path is relevant and necessary at the time of its inclusion.